README.md

```diff
@@ -426,6 +433,165 @@ queue.offer('Furkan').then(function () {
+
+Let’s take a look at the following example.
+
+```javascript
+client.clusterService.on('memberAttributeChanged', function (memberAttributeEvent) {
+    console.log('Member Attribute Changed: The address is', memberAttributeEvent.member.address.toString());
+});
+```
+
+####  4.1.2. Listening for Distributed Object Events
+
+The events for distributed objects are invoked when they are created and destroyed in the cluster. After the events, a listener callback function is called. The type of the callback function should be `DistributedObjectListener`. The parameter of the function is `DistributedObjectEvent` including following fields:
+
+`serviceName`: Service name of the distributed object.
+
+`objectName`: Name of the distributed object.
+
+`eventType`: Type of the invoked event. It can be `created` or `destroyed`.
+
+The following is an example of adding a Distributed Object Listener.
+
+```javascript
+client.addDistributedObjectListener(function (distributedObjectEvent) {
+    console.log('Distributed object event >>> ',
+        distributedObjectEvent.serviceName,
+        distributedObjectEvent.objectName,
+        distributedObjectEvent.eventType
+    );
+}).then(function () {
+    var mapname = 'test';
+    //this causes a created event
+    client.getMap(mapname);
+    //this causes no event because map was already created
+    client.getMap(mapname);
+});
+```
+
+#### 4.1.3. Listening for Lifecycle Events
+
+The Lifecycle Listener notifies for the following events:
+- `starting`: A client is starting.
+- `started`: A client has started.
+- `shuttingDown`: A client is shutting down.
+- `shutdown`: A client’s shutdown has completed.
+
+The following is an example of Lifecycle Listener that is added to config and its output.
+
+```javascript
+var clientConfig = new Config.ClientConfig();
+clientConfig.listeners.addLifecycleListener(function (state) {
+    console.log('Lifecycle Event >>> ' + state);
+});
+
+Client.newHazelcastClient(clientConfig).then(function (hazelcastClient) {
+    hazelcastClient.shutdown();
+});
+```
+
+**Output:**
+
+```
+Lifecycle Event >>> starting
+[DefaultLogger] INFO at ConnectionAuthenticator: Connection to 10.216.1.62:5701 authenticated
+[DefaultLogger] INFO at ClusterService: Members received.
+[ Member {
+    address: Address { host: '10.216.1.62', port: 5701, type: 4 },
+    uuid: 'dc001432-7868-4ced-9161-5649ff6f31fc',
+    isLiteMember: false,
+    attributes: {} } ]
+Lifecycle Event >>> started
+[DefaultLogger] INFO at HazelcastClient: Client started
+Lifecycle Event >>> shuttingDown
+Lifecycle Event >>> shutdown
+
+Process finished with exit code 0
+```
+
+### 4.2. Distributed Data Structure Events
+
+This chapter explains when various events are fired and describes how you can add event listeners on a Hazelcast Node.js client. These events can be categorized as cluster and distributed data structure events.
```

Comment:
No need for this paragraph as it is the same with the intro paragraph of this chapter. Let's remove it.

Reply:
Sorry, It is a mistake. Wouldn't it be good to add `You can add event listeners to Distributed Data Structures.` ?

Reply:
👍